modulo-1-fundamentos/bloco-8-higher-order-functions-do-javascript-es6/dia-1-javascript-es6-introducao-a-higher-order-functions/script03.js

```diff
@@ -0,0 +1,17 @@
+const RIGHT_ANSWERS = ['A', 'C', 'B', 'D', 'A', 'A', 'D', 'A', 'D', 'C'];
+const STUDENT_ANSWERS = ['A', 'N.A', 'B', 'D', 'A', 'C', 'N.A', 'A', 'D', 'B'];
+
+const handleGrading = (masterAnswerSheet, studentAnswerSheet) => {
+  let grade = 0;
+
+  for (i=0; i < masterAnswerSheet.length; i += 1) {
+    if (masterAnswerSheet[i] === studentAnswerSheet[i]) {grade += 1} 
```

Comment:
A título de curiosidade, como o seu if só tem uma linha, vc pode fazer o mesmo esquema de uma arrow function, não precisa colocar entre {}
```suggestion
    if (masterAnswerSheet[i] === studentAnswerSheet[i]) grade += 1
```

Reply:
Show, não sabia